src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -51,6 +51,18 @@ struct result {
   data_iterator::batch data_info;
 };
 
+gl_sarray get_predictions_class(gl_sarray predictions_prob,
```

Comment:
do you want to inline it? or make it a private/protected member function? Default static linkage is not recommended.

Reply:
Oh okay I'll make it private.

Reply:
I think protected is better :-)

Reply:
let's do protected inline. Just put the whole definition to the header file!

Reply:
good!